docs/quickstart/customize.md

```diff
@@ -18,3 +20,24 @@ Updating static files...
 Go to the browser [*https://localhost/*](https://localhost) or refresh the page. And voila! The logo has been changed!
 
 **WARNING**: If you do not see it changing, check in an incognito window, the browser might have cached the logo.
+
+## CSS + Templates
+
+If you want to customize the look and feel of your instance, you will want to customize the css and templates used.
+
+For the templates, you can just put your new Jinja templates in the `templates/` folder and refer to them from the `invenio.cfg` file appropriately.
```

Comment:
Minor: Shall we link to https://invenio-theme.readthedocs.io/en/latest/configuration.html in order to provide the full list of configuration available?

Reply:
Yes, that's more useful. Added.